src/app/containers/ConsentBanner/a11y.test.jsx

```diff
@@ -75,3 +101,41 @@ describe('a11y', () => {
     expect(document.activeElement).toBe(getByText('BBC Brand'));
   });
 });
+
+describe('amp', () => {
+  it('should render a focussable manage cookies heading on AMP', () => {
+    const { container } = render(
+      <AmpBannerWithContext
+        service="pidgin"
+        serviceConfig={pidginServiceConfig}
+        variant="default"
+      />,
+    );
+
+    const manageCookiesHeading = container.querySelector(
+      '#manageCookiesHeading',
+    );
+    manageCookiesHeading.focus();
+
+    expect(document.activeElement).toBe(manageCookiesHeading);
```

Comment:
Were you not able to click the relevant buttons etc to test the focus behaviour as in the canonical test: https://github.com/bbc/simorgh/pull/8986/files#diff-84ca7f8699ea94dbc1f36ebb458987a2fcfbeebc9e6bf7e7a50808f1fa3bc904R99
?

Reply:
Unfortunately not, I tried but it seems that the AMP focus action doesn't work in these tests, even with the necessary scripts rendered